src/event.rs

```diff
@@ -317,6 +317,22 @@ pub enum WindowEvent {
         ///
         /// This value may be NaN.
         delta: f64,
+        velocity: f32,
```

Comment:
Needs doc comment, esp. to explain the difference between `delta` and `velocity` in this context.